examples/misc/agg_buffer_to_array.py

```diff
@@ -7,9 +7,12 @@
 """
 import matplotlib.pyplot as plt
 import numpy as np
+from matplotlib.figure import Figure
 
-# make an agg figure
-fig, ax = plt.subplots()
+# make an agg figure.  If no canvas is specified, will default to
+# `matplotlib.backends.backend_agg.FigureCanvas`
```

Comment:
That statement is not true...

Reply:
:sheep: indeed.  Was getting too tired to be effective last night....

Force-pushed out of existence.